src/pybag/core.pyi

```diff
@@ -295,21 +295,6 @@ class PyLayInstRef:
     def transform(self, xform: Transform) -> None: ...
 
 
-class PyOADatabase:
```

Comment:
Is it ok to delete PyOADatabase from the "with OA" version? I guess users are expected to build the C++ which will generate these .pyi files anyway, so maybe it's ok. In that case, maybe you can even stop git tracking .pyi files, otherwise it will show up in "git status" even though user hasn't changed anything intentionally.

Reply:
I agree. In the next push, I will remove the `.pyi` files and add a comment in the README.